packages/access-client/src/agent.js

```diff
@@ -322,6 +323,10 @@ export class Agent {
       throw new Error('Recover validation failed', { cause: inv })
     }
 
+    if (inv && 'match_phrase' in inv) {
+      opts?.handlePhrase?.(inv.match_phrase)
```

Comment:
I typically I'd call this `onPhrase` to signify that it is an event - you'd maybe call your event handler `handlePhrase` but the option should be the event name (prefixed with "on").

We have prior art in other modules e.g. https://github.com/web3-storage/w3protocol/blob/main/packages/upload-client/src/types.ts#L214 so this would be consistent with them.